docs/workflows/elastic_import.md

```diff
@@ -0,0 +1,109 @@
+# Elastic Import
+
+```eval_rst
++------------------------------+---------+
+| Summary                      |         |
++==============================+=========+
+| Runs on remote worker        | True    |
```

Comment:
It runs on the main VM so this should be False.

Reply:
Ah yeah, thanks for picking that up